client/blocks/product-selector/index.jsx

```diff
@@ -88,11 +102,19 @@ export class ProductSelector extends Component {
 		const { description, optionDescriptions } = product;
 		const purchase = this.getPurchaseByProduct( product );
 
-		if ( ! purchase || ! optionDescriptions || ! optionDescriptions[ purchase.productSlug ] ) {
-			return description;
+		// Description, obtained from a purchased product
+		if ( purchase && optionDescriptions && optionDescriptions[ purchase.productSlug ] ) {
+			return optionDescriptions[ purchase.productSlug ];
+		}
+
+		// Description, obtained from a product that's included in a purchased plan
+		const planProductSlug = this.getProductSlugByCurrentPlan();
+		if ( planProductSlug && optionDescriptions[ planProductSlug ] ) {
```

Comment:
Should we also check if `optionDescriptions` array exists before trying to access `optionDescriptions[ planProductSlug ]`?

I'm asking since we're checking this [6 lines above](https://github.com/Automattic/wp-calypso/pull/37396/files#diff-dfff688a37f93f6dc6effc4f8b83730dR106) and not here.

Reply:
👍 Fixed in https://github.com/Automattic/wp-calypso/pull/37396/commits/a123b5f15c4a6fd0b5321dd995f3328b388678a8.